packages/portfolio-deploy/src/axelar-configs.js

```diff
@@ -1,18 +1,75 @@
+/**
+ * @import {AxelarChain} from '@aglocal/portfolio-contract/src/constants.js';
+ * @import {AxelarConfig, EVMContractAddresses} from '@aglocal/portfolio-contract/src/portfolio.contract.ts';
+ * @import {BaseChainInfo} from '@agoric/orchestration'
+ * @import {EVMContractAddressesMap} from '@aglocal/portfolio-contract/src/type-guards.ts';
+ */
+
+/**
+ * @typedef {object} AxelarChainIdEntry
+ * @property {string} testnet - The Axelar chain ID used in testnet.
+ * @property {string} mainnet - The Axelar chain ID used in mainnet.
+ *
+ */
+
+/**
+ * A mapping between internal AxelarChain enum keys and their corresponding
+ * Axelar chain identifiers for both testnet and mainnet environments.
+ *
+ * This is used by YMax to dynamically switch between environments when interacting
+ * with Axelar-supported chains.
+ *
+ * @type {Record<keyof typeof AxelarChain, AxelarChainIdEntry>}
+ *
+ * @see {@link https://docs.axelar.dev/resources/contract-addresses/testnet/#evm-contract-addresses}
+ * @see {@link https://github.com/axelarnetwork/axelarjs-sdk/blob/f84c8a21ad9685091002e24cac7001ed1cdac774/src/chains/supported-chains-list.ts | supported-chains-list.ts}
+ */
+const AxelarChainIdMap = harden({
```

Comment:
I'd expect `testnet` and `mainnet` to be the top level keys; one of them is selected and passed to the relevant consumer.